README.md

```diff
@@ -128,7 +128,7 @@ constructorIO.trackSearchSubmit(searchTerm: "toothpicks", originalQuery: "tooth"
 constructorIO.trackSearchResultsLoaded(searchTerm: "tooth", resultCount: 789)
 
 // Track when a search result is clicked
-constructorIO.trackSearchResultClick(itemName: "Fashionable Toothpicks", customerID: "1234567-AB", searchTerm: "tooth")
+constructorIO.trackSearchResultClick(itemName: "Fashionable Toothpicks", customerID: "1234567-AB", searchTerm: "tooth", resultID: "179b8a0e-3799-4a31-be87-127b06871de2")
```

Comment:
there is a sectionName argument before resultID, so we should add the sectionName as well so something like `sectionName: "Products", resultID: "179b8a0e-3799-4a31-be87-127b06871de2"`

Reply:
Ah, I guess we missed the section name for the original docs?

Reply:
e8ab7e4